participants-schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Participants Schema",
+  "type": "object",
+  "definitions": {
+    "url": {
+      "type": "string",
+      "pattern": "^(http|https)://[A-Za-z0-9ÄÖÜäöüß\\-_\\./]+"
+    }
+  },
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "pronouns": {
+      "type": "string"
+    },
+    "link": {
+      "$ref": "#/definitions/url"
+    },
```

Comment:
This should be merged with "url" and only link should be allowed.

Reply:
image is an url, too, that's why I extracted this type to a definition

Reply:
image is an url, too, that's why I extracted that type to a definition